packages/core/components/Filters.tsx

```diff
@@ -348,6 +355,7 @@ const Filters = (props: FilterProps) => {
     )
   }
 
+  Filters.NestedDropdown = props => props.nestedDropdown
```

Comment:
what does this do?

Reply:
I believe this creates a Filters.NestedDropdown that passes the props from Filters into it. This is used for all filters and helps with the functionality of the Filters component. To be honest, I am not a 100% sure how it works but, I know with out it, the nested dropdown doesn't work properly.